### PR TITLE
Update file-encryption.html

### PR DESCRIPTION
**Pull request by earonesty**
## Description

Hi, I'm the CTO at Atakama.   We've spent the last few years developing frictionless, password-less multifactor encryption software.   We use a secure DKG to generate a multifactor public/private key pair on the secp256k1 curve, and then use ECIES/AES/HMAC to encrypt files on a mounted drive.   It's free to use, and we have an enterprise version to pay for development

It's great for people that want the protections of encryption, but also want a convenient UX, and don't want more passwords to manage or remember.   

It also provides serious malware protection, since files are not decrypted until they are in-use.   No keylogging can steal your master key - because it doesn't exist, etc.

It's step up, and I really would like your opinion on it.

### CODE REVIEW

File: _includes/legacy/sections/file-encryption.html [hunk 1]
```diff
@@ -65,4 +65,5 @@ <h3>Worth Mentioning</h3>
         tooltip="As Kryptor is still in beta, it may not be stable."
       %}
     </li>
+  <li><a href="https://cryptomator.org/">Atakama</a> - Free, passwordless, multi-factor file encryption. No backdoors, no central control, no registration.</li>
```
Comment (rusty-snake) [inline _includes/legacy/sections/file-encryption.html:68]: Copy&Paste: `https://cryptomator.org/`

Comment (jonaharagon) [reply _includes/legacy/sections/file-encryption.html:68]:
```suggestion
  <li><a href="https://www.atakama.com/">Atakama</a> - Free, passwordless, multi-factor file encryption. No backdoors, no central control, no registration.</li>
```

**Comment by samuel-lucas6** [comment]
@earonesty I've opened a discussion for you. Please see #139.

**Comment by samuel-lucas6** [comment]
@jonaharagon @freddy-m I think this can be closed for the reasons I outlined in the discussion and because there's been no response since this PR was opened.

**Pull request closed without merging**